### PR TITLE
Issue 5 - Remove dependency on five.grok

### LINKED ISSUE #5: Remove dependency on five.grok

### PR DESCRIPTION
**Pull request by rodfersou**
closes #5

### CODE REVIEW

File: src/collective/liveblog/browser/header.py [hunk 1]
```diff
@@ -1,22 +1,11 @@
...
-    grok.name('collective.liveblog.header')
-    grok.context(ILiveblog)
-    grok.layer(IBrowserLayer)
-    grok.require('zope2.View')
-    grok.viewletmanager(IAboveContent)
-
     def available(self):
         """Return True if an image has been defined."""
-        return self.context.image is not None
+        return getattr(self.context, 'image', None) is not None
```
Comment (rodfersou) [inline src/collective/liveblog/browser/header.py]: one time the context was plone portal object and this raises exception for me

Comment (hvelarde) [inline src/collective/liveblog/browser/header.py]: this makes no sense; are you registering the viewlet against the right context? you must use `ILiveblog`.

File: src/collective/liveblog/browser/templates/header.pt [hunk 2]
```diff
@@ -5,11 +5,11 @@
       i18n:domain="collective.liveblog"
       tal:omit-tag="">
 <body tal:omit-tag="">
-  <tal:viewlet condition="viewlet/available">
+  <tal:viewlet condition="view/available">
```
Comment (hvelarde) [inline src/collective/liveblog/browser/templates/header.pt]: are you sure we need this change?

File: src/collective/liveblog/browser/configure.zcml [hunk 3]
```diff
@@ -29,4 +29,40 @@
...
+  <browser:page
+      for="collective.liveblog.interfaces.ILiveblog"
+      name="recent-updates"
+      class="collective.liveblog.browser.recent_updates.RecentUpdates"
+      permission="zope2.View"
+      layer="collective.liveblog.interfaces.IBrowserLayer"
+      template="templates/recent_updates.pt"
+      />
+
+  <browser:viewlet
```
Comment (hvelarde) [inline src/collective/liveblog/browser/configure.zcml]: I think you're missing here the `for` directive

File: src/collective/liveblog/browser/templates/header.pt (hunk 2 above)
Comment (rodfersou) [inline src/collective/liveblog/browser/templates/header.pt]: If you don't want to raise an exception, yes

File: src/collective/liveblog/browser/configure.zcml (hunk 3 above)
Comment (rodfersou) [inline src/collective/liveblog/browser/configure.zcml]: done

**Pull request merged**